i18n/translations.en.json

```diff
@@ -119,7 +119,7 @@
       "password": "Password",
       "newPassword": "Password",
       "confirmPassword": "Confirm Password",
-      "to": "To",
+      "to": "email:john@gmail.com, user:pphetra",
```

Comment:
Shouldn't this be "To" and not directly to @pphetra ?

Reply:
That key in under "placeholder" group. 
<img width="837" alt="Screen Shot 2565-10-18 at 15 06 22" src="https://user-images.githubusercontent.com/1422/196373013-a5f41892-233e-4e7c-b338-d120446ee3aa.png">